### DISCUSSION TITLE
YouTube embed with start time

### DISCUSSION THREAD

**Discussion by AKRking** [steven-tey/novel#462] [Q&A]
Is there any way to embed YouTube video with start time? can't figure out how to modify yt extension of tiptap

**Comment by kamyarkaviani** [comment]
See https://developers.google.com/youtube/player_parameters. Just pass in the query param start in number of seconds since the start of the video, eg, https://www.youtube.com/embed/bItvhlRVQhM?start=125.

**Reply by AKRking** [reply]
yes I attempted to do it first but it didn't work, later on I had to modify some of the code to make it work

**Reply by AKRking** [reply]
https://github.com/steven-tey/novel/discussions/499
this works but gives hydration error
